### ISSUE TITLE
Design Feedback und Kleinigkeiten

### ISSUE THREAD

**Issue by annikabock** [ubilabs/esa-climate-from-space#1671]
- [x] Font color for story titles should be turquoise - or did you change this because of accessibility?
- [x] when scrolling through the stories, the globe may not react immediately but with a little "delay", so that it stays where it is and as soon as the user stops at one story the globe starts turning
- [x] story titles should be elipsed when too long and only be shown with line break when in focus
